scripts/build_scenario1.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from jinja2 import Environment, FileSystemLoader
+
+# Définir le répertoire contenant le template
+template_dir = '/home/chausser/test/'
+
+
+
+# Configuration de Jinja2 pour charger le template
+env = Environment(loader=FileSystemLoader(template_dir))
+template = env.get_template('scenario1.yaml.j2')
+for missing in range(1,7):
```

Comment:
this should be configurable variables at the top of the script, i.e.

MISSING_RATE = range(1, 7)

etc for easy change